### PR TITLE
Twin tools

### PR DESCRIPTION
**Pull request by schoondw**
# Description
Twin tools adds a menu *Twin* containing tools for twin system users. Currently, it contains one tool for obtaining twin calibration parameters using a rigid body.

# Use case
Alternative twin calibration method for applications in which it is difficult to perform the twin wand calibration.

# Testing
Tested with twin measurements with an underwater and above water system. Tested for a variety of calibrations (different alignments of coordinate systems) and rigid body poses.

### CODE REVIEW

File: tools/twin_tools.py [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+menu_priority = 1
+
+def add_menu():
+    if have_numpy:
+        # - Add commanads and set up menu
+        add_my_commands()
+        setup_my_menu()
+    else:
+        print(f"You need Numpy installed in Python to use the Refine Rigid Body menu.")
```
Comment (ulvs) [inline tools/twin_tools.py]:
```suggestion
        print(f"You need Numpy installed in Python to use the Twin menu.")
```

**Review by ulvs — CHANGES_REQUESTED** [review]
Nice structured code.

Just fix the copy paste error.

**Review by ulvs — APPROVED** [review]

File: tools/twin_tools.py [hunk 2]
```diff
@@ -0,0 +1,184 @@
+"""twin_tools.py: Tools for twin system
+
+General info
+Tools for twin system. Currently, one method is included for twin calibration
+based on 6DOF rigid bodies with a fixed relationship that can be used as an alternative
+to the twin wand calibration, for example for application in which it is difficult to
+access the volume with a wand.
+
+Description of menu
+This scripts adds a menu "Twin" to QTM the QTM menu bar.
```
Comment (Henrik-Norgren) [inline tools/twin_tools.py:10]: nitpick: Grammar

File: tools/twin_tools.py [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+def add_my_commands():
+    """Function for adding new commands used in this script to QTM"""
+
+    # Add command for display of script doc string
+    command_name = "disp_twin_tools_doc"
+    qtm.gui.add_command(command_name)
+    qtm.gui.set_command_execute_function(command_name, lambda:(print(__doc__)))
+
+    # Add command for updating the rigid body list for the
```
Comment (Henrik-Norgren) [inline tools/twin_tools.py:157]: nitpick: Comment ends abruptly.

**Review by Henrik-Norgren — APPROVED** [review]

**Pull request merged**